API.md

```diff
@@ -1198,6 +1200,51 @@ const schema = Joi.object({
 });
 ```
 
+#### `date.greater(date)`
+
+Specifies that the value must be greater than `date`.
+
+```js
+const schema = Joi.date().greater('1-1-1974');
+```
+
+Notes: `'now'` can be passed in lieu of `date` so as to always compare relatively to the current date, allowing to explicitly ensure a date is either in the past or in the future.
+
+```js
+const schema = Joi.date().greater('now');
+```
+
+It can also be a reference to another field.
+
+```js
+const schema = Joi.object({
+  from: Joi.date().required(),
+  to: Joi.date().greater(Joi.ref('from')).required()
+});
+```
+
+#### `date.less(date)`
+
+Specifies that the value must be less than `date`.
+
+```js
+const schema = Joi.date().less('12-31-2020');
+
```

Comment:
It's been a year now and nobody noticed that the docs are broken because of a missing code block ending :rofl:

Reply:
@legraphista Or no one cares :man_shrugging: It's fixed.

Reply:
@Marsup unfortunately Markdown -> HTML converters do care :confused: 
Still one left https://github.com/hapijs/joi/pull/2171